lesson05/assignment/index.html

```diff
@@ -9,7 +9,56 @@
     </head>
     <body>
         <main class="main">
-          
+            <div class="grid ">
+                <div class="item" tabindex="1">
+                  <div class="box">
+                    <div class="header">Header</div>
+                  </div>
+                </div>
+                <div class="item" tabindex="2">
+                  <div class="box">
+                    <div class="nav_item">Nav Item</div>
```

Comment:
With our nav, we'd need the elements to be `<a>` so we can give them an href to goto. We can just use `#` as a placeholder value.